jpa/eclipselink.jpa.test.jse/src/it/java/org/eclipse/persistence/jpa/test/sequence/TestIdentityGeneration.java

```diff
@@ -95,6 +96,29 @@ public void testPersistWithSecondaryTables() {
         }
     }
 
+    @Test
+    public void testPersistWithSecondaryTables2() {
```

Comment:
Would it make sense to add a reference to a bug this test is covering either as a comment or as a link here?

Reply:
@lukasj Sure! How is that?